### PR TITLE
testing/irtest: Remove an unused function

### PR DESCRIPTION
**Pull request by yamt**
## Summary
testing/irtest: Remove an unused function
## Testing
build-tested

### CODE REVIEW

File: testing/irtest/cmd.cpp [hunk 1]
```diff
@@ -69,20 +69,6 @@ static void print_cmd(const cmd *cmd)
   printf(")\n");
 }
 
-static int print_cmd(const char *name)
```
Comment (xiaoxiang781216) [inline testing/irtest/cmd.cpp:72]: called at line 76?

Comment (yamt) [reply testing/irtest/cmd.cpp:72]: it's another print_cmd at line 61, not this one.

Comment (xiaoxiang781216) [reply testing/irtest/cmd.cpp:72]: Ok, it's overloaded.

**Review by xiaoxiang781216 — APPROVED** [review]

File: testing/irtest/cmd.cpp (hunk 1 above)
Comment (xiaoxiang781216) [reply testing/irtest/cmd.cpp:72]: I am curious why CI can't catch this warning.

**Review by xiaoxiang781216 — APPROVED** [review]

**Review by jerpelea — APPROVED** [review]

**Pull request merged**